script.js

```diff
@@ -0,0 +1,127 @@
+  },
+  {
+    id: 3,
+    pic: 'https://images.pexels.com/photos/8482835/pexels-photo-8482835.jpeg?',
+    altText: 'featured speaker image',
+    nameOfSpeaker: 'Roger Twesigye',
+    designation: 'CEO hollywood homes.',
+    history:
+      'Vast years of experience in interior, nationally and internationally',
+  },
+  {
+    id: 4,
+    pic: 'https://cdn.xxl.thumbs.canstockphoto.com/realtor-happy-realtor-woman-is-showing-home-for-sale-sign-and-keys-stock-photos_csp27243435.jpg',
+    altText: 'featured speaker image',
+    nameOfSpeaker: 'Nick',
+    designation: 'Founder of building homes',
+    history:
+      'Vast years of experience in Real Estate, nationally and internationally',
+  },
+  {
+    id: 5,
+    pic: 'https://images.pexels.com/photos/864994/pexels-photo-864994.jpeg?auto=compress&cs=tinysrgb&w=600.png',
+    altText: 'featured speaker image',
+    nameOfSpeaker: 'Racheal Mbabazi',
+    designation: 'CEO of lets Build.',
+    history:
+      'Six years of experience in interior design, nationally and internationally',
+  },
+  {
+    id: 6,
+    pic: 'https://images.pexels.com/photos/7642009/pexels-photo-7642009.jpeg?auto=compress&cs=tinysrgb&w=600.png',
+    altText: 'featured speaker image',
+    nameOfSpeaker: 'Phillip Musiime',
+    designation: 'Founder of Tomorrow World',
+    history:
+      'Vast years of experience in real estate, nationally and internationally.',
+  },
+];
+
+const speakerContainer = document.createElement('section');
+speakerContainer.setAttribute('class', 'featured-speakers');
+speakerContainer.setAttribute('id', 'speakers');
+data.forEach((speakerPara) => {
+  const speakerSection = document.createElement('div');
+  speakerSection.setAttribute('class', 'speaker-section');
+  speakerSection.setAttribute('id', 'each-speaker');
+
+  const imageSection = document.createElement('div');
+  imageSection.setAttribute('class', 'speaker-image-section');
+  const image = document.createElement('img');
+  image.setAttribute('class', 'speaker-image');
+  image.setAttribute('src', speakerPara.pic);
+  image.setAttribute('alt', speakerPara.altText);
+
+  imageSection.appendChild(image);
+  speakerSection.appendChild(imageSection);
+
+  const contentSection = document.createElement('div');
+  contentSection.setAttribute('class', 'speaker-details');
+  contentSection.setAttribute('id', 'about-speaker');
+
+  const speakerName = document.createElement('h3');
+  speakerName.setAttribute('class', 'speaker-name');
+  speakerName.textContent = speakerPara.nameOfSpeaker;
+  contentSection.appendChild(speakerName);
+
+  const speakerDesignation = document.createElement('p');
+  speakerDesignation.setAttribute('class', 'speaker-designation');
+  speakerDesignation.textContent = speakerPara.designation;
+  contentSection.appendChild(speakerDesignation);
+
+  const dotText = document.createElement('p');
+  dotText.setAttribute('id', 'dots');
+  dotText.textContent = '.........';
```

Comment:
As recommend by the previous reviewer, kindly adjust the speakers' section to match the given design. Make the dividing rule in the speaker cards to be like it is on the design. It's a continuous line and not dotted.

Also italicise and reduce the font size of the title of the speaker on red text and add the mini background behind the speakers' pictures as it appears on the design.



Your design:
![image](https://user-images.githubusercontent.com/20554583/191715579-df39e47c-9ad4-4a79-b0b0-d7f927f79156.png)


Design guideline:
![image](https://user-images.githubusercontent.com/20554583/191716056-64878b85-7910-4ac0-8745-dc31d1326127.png)